### ISSUE TITLE
Some mp3's crash DCA it seems..

### ISSUE THREAD

**Issue by karbowiak** [teamreflex/DiscordPHP#111]
```PHP Fatal error:  Uncaught Discord\Exceptions\OutdatedDCAException: You are using an outdated version of DCA. Please make sure you have the latest version from https://github.com/bwmarrin/dca in /storage/dev/Sovereign/vendor/team-reflex/discord-php/src/Discord/Voice/VoiceClient.php:780
Stack trace:
#0 [internal function]: Discord\Voice\VoiceClient->Discord\Voice\{closure}(Resource id #1922, Object(React\EventLoop\StreamSelectLoop))
#1 /storage/dev/Sovereign/vendor/react/event-loop/src/StreamSelectLoop.php(232): call_user_func(Object(Closure), Resource id #1922, Object(React\EventLoop\StreamSelectLoop))
#2 /storage/dev/Sovereign/vendor/react/event-loop/src/StreamSelectLoop.php(201): React\EventLoop\StreamSelectLoop->waitForStreamActivity(4576.9214630127)
#3 /storage/dev/Sovereign/vendor/team-reflex/discord-php/src/Discord/WebSockets/WebSocket.php(917): React\EventLoop\StreamSelectLoop->run()
#4 /storage/dev/Sovereign/src/sovereign.php(449): Discord\WebSockets\WebSocket->run()
#5 /storage/dev/Sovereign/Sovereign(60): Sovereign\S in /storage/dev/Sovereign/vendor/team-reflex/discord-php/src/Discord/Voice/VoiceClient.php on line 780```

So yeah, DCA seems to be outdated, maybe an update is in order? :P

**Comment by uniquoooo** [comment]
I need to replace the DCA version shipped with DiscordPHP, I will do this when I get some time but in the meantime you can compile this and replace the DCA file in the `bin` folder: https://github.com/uniquoooo/dca

**Comment by karbowiak** [comment]
That did it, awesome! :D

**Comment by karbowiak** [comment]
Fixed with 673825b2539c7ffedb1aa38f06d1825da9a925a1
